### PR TITLE
[FIX] html_editor: remove correctTripleClick in delete_plugin

### PR DESCRIPTION
**Pull request by Mtaylorr**
Issue:
======
Traceback raised when delete_backward

Steps to reproduce the issue:
=============================
- Put the cursor in a cell in the middle of the table
- Press shift and arrow left/up until you have the selection outside of the table
- Delete backward

Origin of the issue:
====================
`correctTripleClick` moves the endContainer to be  the `br` element in the cell just before where we put the cursor. Since we remove fake brs in deleteRange and then we removeNodes, we have the endContainer is not connected anymore and raises an error when we want to get it's `childNodeIndex`.

It's safe to remove the function `correctTripleClick` since the correction is now handled by `selection_plugin`.

### CODE REVIEW

File: addons/html_editor/static/tests/unbreakable/delete.test.js [hunk 1]
```diff
@@ -263,29 +263,22 @@ describe("backward", () => {
...
-            test("should delete last character of paragraph, ignoring the selected paragraph break leading to an unbreakable (1)", async () => {
-                await testEditor({
-                    contentBefore: `<p>ab[c</p><p class="oe_unbreakable">]def</p>`,
-                    // This type of selection (typically done with a triple
-                    // click) is "corrected" before remove so triple clicking
-                    // doesn't remove a paragraph break.
-                    stepFunction: deleteBackward,
-                    contentAfter: `<p>ab[]</p><p class="oe_unbreakable">def</p>`,
-                });
+        test("should delete last character of paragraph and the empty unbreakable (1)", async () => {
```
Comment (cammarosano) [inline addons/html_editor/static/tests/unbreakable/delete.test.js:266]: There's no empty unbreakable here. I guess it comes from the test below...
I suggest "should delete last character of paragraph but not merge it with unbreakable"

File: addons/html_editor/static/tests/unbreakable/delete.test.js [hunk 2]
```diff
@@ -263,29 +263,22 @@ describe("backward", () => {
...
-            test("should delete last character of paragraph, ignoring the selected paragraph break leading to an unbreakable (2)", async () => {
-                await testEditor({
-                    contentBefore: `<p>ab[c</p><p class="oe_unbreakable">]<br></p><p>def</p>`,
-                    // This type of selection (typically done with a triple
-                    // click) is "corrected" before remove so triple clicking
-                    // doesn't remove a paragraph break.
-                    stepFunction: deleteBackward,
-                    contentAfter: `<p>ab[]</p><p class="oe_unbreakable"><br></p><p>def</p>`,
-                });
+        test("should delete last character of paragraph and the empty unbreakable (2)", async () => {
```
Comment (cammarosano) [inline addons/html_editor/static/tests/unbreakable/delete.test.js:274]: ...then you can remove the "(2)" here.
But actually I would also suggest "should delete last character of paragraph and fully selected empty unbreakable"

**Comment by Mtaylorr** [comment]
Thanks for the review @cammarosano , I fixed the comments

**Comment by ged-odoo** [comment]
robodoo rebase

**Comment by ged-odoo** [comment]
robodoo r+

**Comment by ged-odoo** [comment]
robodoo rebase-ff

**Pull request closed without merging**